LiskHQ/lisk-desktop
# Added status for hw wallet

### What was the problem?

This PR resolves #4882 except for error handling

This PR fixes these issues.
1. Implement hardware wallet status functionality and 
2. event connection 

Error handling will be added in another PR

### How was it solved?
- Polling HW wallet to see if the Lisk app is open on device. 
- Updating @zondax/ledger-lisk API

### How to test?
- update dependencies so you are using latest zondax lib when testing
- Verify that the HW status in the top menu is showing green when Lisk app is open and yellow when it is closed but the ledger is connected to the computer.
- Verify that HW accounts are loaded automatically in our switch account flow, no need to page refresh or manually opening the modal to refresh accounts anymore. 
- Send transaction should work

### How was it tested?

Manually
